CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+## THIS SHOULD ALWAYS BE FIRST ##
+* @BitGo/btc-team
+
+.github @BitGo/btc-team @BitGo/infra @BitGo/velocity
+renovate.json @BitGo/btc-team @BitGo/infra @BitGo/velocity
```

Comment:
sounds like velocity doesn't want/need to be involved so we can remove them.  Otherwise changes are looking better

Reply:
adjusted.